MAUI/TabView/Tab-Item-Customization.md

```diff
@@ -221,6 +221,31 @@ The image will be placed to the right side of the text horizontally.
 
 ![Tab Image Position Right](images/Image-Position-Right.png) 
 
+## Image Text Spacing
+
+The `ImageTextSpacing` property in SfTabItem allows to provide spacing between the image and the text of the tab item.
```

Comment:
The `ImageTextSpacing` property in SfTabItem allows for the provision of spacing between the image and the text of the tab item.

Reply:
Addressed the review changes